<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор выездного листа</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/edit_user</normaloff>:/edit_user</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Выездной лист</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="Label">
         <property name="text">
          <string>Фамилия</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leFamiliia"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Label_2">
         <property name="text">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leImia"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Label_3">
         <property name="text">
          <string>Отчество</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leOtchestvo"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Label_4">
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="Label_5">
         <property name="text">
          <string>Пол</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbPol"/>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="dtBirth"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Информация о поездках за границу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbAddRecord">
         <property name="toolTip">
          <string>Добавить поездку</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/add</normaloff>:/add</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRemoveRecord">
         <property name="toolTip">
          <string>Удалить поездку</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/remove</normaloff>:/remove</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leFamiliia</tabstop>
  <tabstop>leImia</tabstop>
  <tabstop>leOtchestvo</tabstop>
  <tabstop>dtBirth</tabstop>
  <tabstop>cbPol</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pbAddRecord</tabstop>
  <tabstop>pbRemoveRecord</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
